extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -319,13 +325,14 @@ export class Bind {
       match = this.compareStringArrays_(initialValue, classes);
     } else {
       const attribute = element.getAttribute(property);
+      initialValue = attribute;
       if (typeof expectedValue === 'boolean') {
-        initialValue = !!attribute;
+        // Boolean attributes return values of either '' or null.
+        match = (expectedValue && initialValue === '')
```

Comment:
I think we should allow `"true"` and `"false"` as well, else we'd disallow expressions like `[attr]="true"`.